platform/ios/src/MGLMapView.h

```diff
@@ -1336,6 +1338,18 @@ MGL_EXPORT IB_DESIGNABLE
  */
 - (void)selectAnnotation:(id <MGLAnnotation>)annotation animated:(BOOL)animated;
 
+/**
+ Selects an annotation and displays its callout view.
+ 
+ Note that a selection initiated by a single tap gesture is always animated.
+ 
+ @param annotation The annotation object to select.
+ @param animated If `YES`, the annotation and callout view's selection and presentation are animated. This parameter does not control whether the map moves.
+ @param expose If `YES` and the annotation is not visible or is partially visible, the map is panned so that the annotation and its callout are brought into view. The annotation is *not* centered within the viewport.
```

Comment:
`animated` is a past participle while `expose` is in the imperative. `exposed` would be more consistent but less intuitive, suggesting that a different verb would be better suited here. “Show” would at least be kind of consistent with `-showAnnotation:animated:`, but not really, because “animated” in that case controls panning.